docs/pages/versions/unversioned/sdk/system-ui.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SystemUI
+sourceCodeUrl: 'https://github.com/expo/expo/tree/master/packages/expo-system-ui'
+---
+
+import APISection from '~/components/plugins/APISection';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight'
+import InstallSection from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+
+**`expo-system-ui`** provides access to the underlying System UI features such as changing the Android Navigation Bar's color or visibility, adjusting the app's theme, making the iOS Home Indicator sticky, and more.
+
+<PlatformsSection android emulator ios simulator />
+
+## Installation
+
+<InstallSection packageName="expo-system-ui" />
+
+## API
+
+```js
+import { SystemUI } from 'expo-system-ui';
+```
+
+<APISection packageName="expo-system-ui" apiName="SystemUI" />
```

Comment:
It looks like the docs data JSON file is missing. Similar to the TS build files, this file need to be regenerated on the source TypeScript files changes, there is a plan to hook it in some way into the current workflow, but for now the command need to be run manually:
```
et gdad -p expo-system-ui
```

Also, ATM docs auto generation did not support `@platform` annotations, but the support will be added in the future. It would be nice if you can run the docs locally and verify that API looks good in there. 🙂